lib/codeowners/checker.rb

```diff
@@ -74,6 +78,21 @@ def useless_pattern
       end
     end
 
+    def invalid_owner
+      return [] unless @validate_owners
+
+      codeowners.select do |line|
+        next unless line.pattern?
+
+        line.owners.any? do |owner|
+          unless owners_list.include?(owner)
+            @when_new_owner&.call(line, owner)
+            true
+          end
+        end
```

Comment:
I think we can subtract one list from another:
```suggestion
        missing = line.owners - owners_list
        missing.each{|owner| @when_new_owner&.call(line, owner)}
        missing.any?
```

Reply:
The multiline suggestion still not working.